### PR TITLE
Load images created with the Image constructor through the proxy (close #1087)

### LINKED ISSUE #1087: Canvas rendered incorrectly after processing
Reproduced in playground. 
Page: http://photo-mosaic-service.bluerobot.com/7683410f-0803-11e7-8bd8-42010af003a4/index.html
See https://github.com/DevExpress/testcafe/issues/1398 for details

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: src/client/sandbox/node/window.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default class WindowSandbox extends SandboxBase {
         this.messageSandbox = messageSandbox;
 
         this.UNCAUGHT_JS_ERROR_EVENT = 'hammerhead|event|uncaught-js-error';
+        this.IMAGE_FORCE_PROXY_SRC_FLAG = 'hammerhead|image|force-proxy-src-flag';
```
Comment (miherlosev) [inline src/client/sandbox/node/window.js]: Difficult to read in conditions.
I propose to rename this to `FORCE_PROXY_SRC_FOR_IMAGE_FLAG`

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: src/client/sandbox/node/element.js [hunk 2]
```diff
@@ -187,8 +187,11 @@ export default class ElementSandbox extends SandboxBase {
                                        urlUtils.getProxyUrl(value, { resourceType, charset: elCharset });
                 }
             }
-            else if (value && !isSpecialPage && !urlUtils.parseProxyUrl(value))
-                args[valueIndex] = urlUtils.resolveUrlAsDest(value);
+            else if (value && !isSpecialPage && !urlUtils.parseProxyUrl(value)) {
+                args[valueIndex] = el[this.nodeSandbox.win.FORCE_PROXY_SRC_FOR_IMAGE_FLAG] ?
```
Comment (churkin) [inline src/client/sandbox/node/element.js]: `FORCE_PROXY_SRC_FOR_IMAGE_FLAG` --> `FORCE_PROXY_SRC_FOR_IMAGE`

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

**Comment by LavrovArtem** [comment]
ping @churkin

**Pull request merged**